gradio/components.py

```diff
@@ -52,9 +52,7 @@ def get_template_context(self):
         """
         :return: a dictionary with context variables for the javascript file associated with the context
         """
-        return {
-            "label": self.label,
-        }
+        return {"name": self.__class__.__name__.lower(), "label": self.label}
```

Comment:
Why did we add name to get_template_context? There was no in Input or Output.

Reply:
Yes but both `InputComponent` and `OutputComponent` had this line:

https://github.com/gradio-app/gradio/blob/1a4302ce883986a5be377c74369ac0fdc45ce307/gradio/inputs.py#L107

And the parent `Component` class would add it:

https://github.com/gradio-app/gradio/blob/1a4302ce883986a5be377c74369ac0fdc45ce307/gradio/component.py#L25